application/models/Question.php

```diff
@@ -1025,6 +1022,14 @@ public function search()
         return $dataProvider;
     }
 
+    /** @inheritdoc */
+    public function scopes()
+    {
+        return array(
+            'primary' => array('condition' => "parent_qid = 0"),
```

Comment:
When is this scope applied? It won't break any other queries?

Reply:
> When is this scope applied? It won't break any other queries?

On demand only : Question::model()->primary()

Reply:
> When is this scope applied? It won't break any other queries?

On demand only : `Question::model()->primary() `

Reply:
Alright, great :+1: